internal/credsretriever/refreshing_cache.go

```diff
@@ -76,9 +76,8 @@ var (
 )
 
 const (
-	defaultActiveRequestRetries  = 16
-	defaultActiveRequestWaitTime = 250 * time.Millisecond
-	defaultActiveRequestInterval = 1 * time.Second
+	defaultActiveRequestRetries  = 9
+	defaultActiveRequestWaitTime = 200 * time.Millisecond
```

Comment:
How was the retry count determined?

Reply:
we have a hard timeout of credential provider in boto client, https://github.com/boto/botocore/blob/develop/botocore/utils.py#L3032